messages/models.tsp

```diff
@@ -209,4 +226,139 @@ model MessageFileObject {
 
   /** The ID of the [message](/docs/api-reference/messages) that the [File](/docs/api-reference/files) is attached to. */
   message_id: string;
+}
+
+/** Represents a message delta i.e. any changed fields on a message during streaming. */
+model MessageDeltaObject {
+  /** The identifier of the message, which can be referenced in API endpoints. */
+  id: string;
+
+  /** The object type, which is always `thread.message.delta`. */
+  object: "thread.message.delta";
+
+  /** The delta containing the fields that have changed on the Message. */
+  delta: {
+    /** The entity that produced the message. */
+    role?: "user" | "assistant";
+
+    /** The content of the message as an array of text and/or images. */
+    content?: MessageDeltaContent[];
+
+    /**
+     * A list of [file](/docs/api-reference/files) IDs that the assistant should use.
+     * Useful for tools like retrieval and code_interpreter that can access files.
+     */
+    @minItems(1)
```

Comment:
I wonder: Is it correct to say @minItems(1) when the default value is an empty array?